wsi/drm/drmdisplay.cpp

```diff
@@ -585,11 +585,16 @@ bool DrmDisplay::CommitFrame(
     *previous_fence_released = true;
   }
 #endif
-
+  static int FirstFrame = 1;
```

Comment:
static bool isFirstFrame = true?

Reply:
Actually this firstFrame logic is fore debug right
We should remove it for cleancode

Reply:
Done, removed the FirstFrame indicator.